modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/services/wstep/impl/CertificateEnrollmentServiceImpl.java

```diff
@@ -120,7 +116,23 @@ public void requestSecurityToken(String tokenType, String requestType,
                 Element element = (Element) headerElement.getObject();
                 headerBinarySecurityToken = element.getFirstChild().getNextSibling().getFirstChild().getTextContent();
             }
+            if (nodeName.equals(PluginConstants.TO)) {
```

Comment:
As a best practice, always compare variable values against known constants to avoid redundant null checks.